README.md

```diff
@@ -1 +1,186 @@
-# MSalyhu_portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+  <img src="./dist/img/my-profile.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Muhammad Salyhu's Portfolio</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+**This is the portfolio for Muhammad Aminu Salihu as a Software Developer**
```

Comment:
- [ ] Please 🙏🏽 add the title in the main header, your Readme should look like the template for professional documentation 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/234560406-536e9b55-11ba-46d3-8cc5-db8922caf398.png)